JSDemos/Demos/Autocomplete/Overview/description.md

```diff
@@ -1 +1,42 @@
-The Autocomplete component is&nbsp;a&nbsp;textbox that provides suggestions while a&nbsp;user types into&nbsp;it.
\ No newline at end of file
+Autocomplete is a text box that displays suggestions while a user types into it. This demo shows how you can customize the Autocomplete component and how to handle its value change.
+
+## Bind Autocomplete to Data
+
+Use one of the following properties to supply data for the component:
+
+* [items[]](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/)             
+Accepts a local data array.
+
+* [dataSource](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#dataSource)            
+Accepts a local data array, a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or one of DevExtreme data stores. In this demo, every Autocomplete component is bound to a local array, except the "Custom Item Template and Data Source Usage" component that uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+
+The [value](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#value) property stores the selected item. You can use the same property to select an item programmatically, as shown in the "Disabled" Autocomplete component.
+
+If the data source contains objects, you should set the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property. It accepts a data field name that uniquely identifies each data object. In this demo, **valueExpr** is set only for the "Custom Item Template and Data Source Usage" Autocomplete because others are bound to arrays of primitive values.
```

Comment:
`you should set the [valueExpr] property` Set it to what?

Reply:
I'll replace "set" with "specify"